Library/Formula/vegeta.rb

```diff
@@ -0,0 +1,26 @@
+require "formula"
+require "language/go"
+
+class Vegeta < Formula
+  homepage "https://github.com/tsenart/vegeta"
+  url "https://github.com/tsenart/vegeta/archive/v5.5.2.zip"
```

Comment:
` tar.gz ` rather than ` .zip ` is standard when using Github sources. The audit should flag this up, I believe.